prime-router/build.gradle.kts

```diff
@@ -581,7 +591,7 @@ jooq {
                                     // A Java regex matching fully-qualified columns, attributes, parameters. Use the pipe to separate several expressions.
                                     // If provided, both "includeExpressions" and "includeTypes" must match.
                                     .withIncludeExpression("action_log.detail")
-                                    .withIncludeTypes("JSONB"),
+                                    .withIncludeTypes("JSONB")
```

Comment:
You and Jimmy appear to have opposite linting conditions for the trailing comma.

Reply:
Yes, different ktlint versions are being used.  I am not using the one set by the gradle plugin, which is used in the build.